packages/manager/modules/pci/src/projects/project/notebooks/add/add.controller.js

```diff
@@ -24,4 +119,41 @@ export default class {
   refreshMessages() {
     this.messages = this.messageHandler.getMessages();
   }
+
+  onNotebookSubmit() {
+    const { selected, nbResources } = this.notebook;
+    const { editor, framework, resource } = selected;
+    const flavorName = resource.flavor.name;
+
+    this.trackNotebooks(`config_create_notebook#${editor.id}#`);
```

Comment:
You will have to remove all `#`s from the tracking ids. There were in the doc just to indicate that the value between them needs to be replaced

Reply:
done, thanks